### ISSUE TITLE
Add zen coderdojo events

### ISSUE THREAD

**Issue by sandrobonazzola** [GaruGaru/Coderdojo-Events#1]
CoderDojo Foundation recommend to use zen platform instead of eventbrite for new events. It would be nice to add support for zen to the app allowing to see the new events as well.

**Comment by GaruGaru** [comment]
Does zen platform provide some kind of apis ?

**Comment by sandrobonazzola** [comment]
Yes, documentation is available here: https://zen.coderdojo.com/documentation
